kroxylicious-filter-api/pom.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>kroxylicious-filter-api</artifactId>
+    <packaging>jar</packaging>
```

Comment:
We should set a version here as this is not going to be the 1.0 api. Or rather we shouldn't be committing to this being the 1.0 API yet with all the API stability guarantees that implies.

Alternatively we should change the version number of the parent pom. Though I'm less sure we want to have everything move with the same version number going forward.

Reply:
Yep, the api should be versioned independently of the other kroxy code (implies we need a stable kroxy-futures module, or to move off kroxy's future implementation)